index.html

```diff
@@ -186,7 +186,7 @@ <h4 id="attribution"></h4>
 	var at = document.getElementById("attribution");
 	var n = parseInt(1+Math.random()*13);
 	horse.src = horsyHorse.horse;
-	at.innerHTML = "Picture attribution: " + horsyHorse.attribution;
+	at.innerHTML = horsyHorse.attribution === "N/A" ? "📸: " + horsyHorse.attribution : "📸: Horse";
```

Comment:
Does that utf-sign render properly? Doesn't render at all in github/chromium....